jest.config.js

```diff
@@ -1,13 +1,32 @@
 process.env.TZ = "GMT";
 module.exports = {
-  preset: "ts-jest",
-  testEnvironment: "node",
-  roots: ["<rootDir>/src", "<rootDir>/tests"],
-  setupFiles: ["jest-plugin-context/setup"],
-  moduleFileExtensions: ["js", "ts"],
-  testResultsProcessor: "jest-sonar-reporter",
-  transform: {
-    "^.+\\.tsx?$": "ts-jest",
-  },
-  testMatch: ["**/*.*Test.ts"],
-};
+  projects: [
+    {
+      displayName: "UNIT",
+      testMatch: ["<rootDir>/tests/**/*.unitTest.ts"],
+      preset: "ts-jest",
+      testEnvironment: "node",
+      roots: ["<rootDir>/tests"],
+      setupFiles: ["jest-plugin-context/setup"],
+      moduleFileExtensions: ["js", "ts"],
+      testResultsProcessor: "jest-sonar-reporter",
+      transform: {
+        "^.+\\.tsx?$": "ts-jest",
+      },
+    },
+    {
+      displayName: "INTEGRATION",
+      testMatch: ["<rootDir>/tests/**/*.intTest.ts"],
+      preset: "ts-jest",
+      testEnvironment: "node",
+      roots: ["<rootDir>/tests"],
+      setupFiles: ["jest-plugin-context/setup"],
+      moduleFileExtensions: ["js", "ts"],
+      testResultsProcessor: "jest-sonar-reporter",
+      transform: {
+        "^.+\\.tsx?$": "ts-jest",
```

Comment:
So I was tired when I talked about this! this needs to be come .ts and all the files renamed need to go back to .ts...

.tsx is a react file

Reply:
Resolving this as the changes are approved by @naathanbrown